app/models/filters.rb

```diff
@@ -4,6 +4,7 @@ def call
       [
         {
           'key' => 'upcoming_statistics',
+          'value' => 'upcoming_statistics',
```

Comment:
Why do we need a `value` attribute that contains the same value as the `key` attribute? Can't we reuse `key`?

Reply:
There is a reason for this, but I can't remember what it is at the moment. I'll take a look this arvo and see what it is.

Reply:
We use `key` in one place and `value` in another. I'll do a little refactor to remove duplication. Thanks 👍

Reply:
Thank you @hannako !